src/xcode

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+error(){
+  red="\e[1;31m"
+  reset="\e[0m"
+  printf "${red}%s${reset}\n" "$1"  >&2
+}
+
+info(){
+  printf "%s\n" "$1"
+}
+
+available_commands=$( \
+  echo "$PATH" \
+  | tr ":" "\n" \
+  | xargs -I {} sh -c 'test -d {} && echo {}' \
+  | xargs -I {} find {} -maxdepth 1 -perm +111 -iname "xcode-*" \
```

Comment:
Why `iname` instead of `name`? The code on line 43 checks against `xcode` (lowercase), so I think having a command named `XCoDe-foo` in the path wouldn't work in any case.